package/libpiksi/libpiksi/src/sbp_tx.c

```diff
@@ -16,7 +16,16 @@
 
 #include <libpiksi/sbp_tx.h>
 
-#define SBP_FRAME_SIZE_MAX 264
+#define SBP_START_BYTE_LEN 1
```

Comment:
Not really about this PR, but why isn't libsbp offering these constants?

Reply:
It probably should!!! Instead we define a "payload max" and a "framing max" values everywhere.

Reply:
https://swift-nav.atlassian.net/browse/ESD-1110

Reply:
👍